src/vec2.rs

```diff
@@ -86,6 +111,23 @@ impl Vec2 {
     }
 
     /// Magnitude squared of vector.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use kurbo::Vec2;
+    /// let v = Vec2::new(3.0, 4.0);
+    /// assert_eq!(v.hypot2(), 25.0);
+    /// ```
+    ///
+    /// # See also
+    ///
+    /// * [`Point::distance_squared`]
+    /// * [`Vec2::dot`]
+    /// * [`Vec2::hypot`]
+    /// * [`Vec2::length`]
+    /// * [`Vec2::length_squared`]
+    /// * [`Vec2::normalize`]
     #[inline]
     pub fn hypot2(self) -> f64 {
```

Comment:
I love that this is `hypot2` whereas the point method is `distance_squared` :)

Reply:
Yes. I find a lot of the naming in `kurbo` to be confusing.